skonfig/__init__.py

```diff
@@ -17,6 +17,9 @@ def run():
         from skonfig import __version__
         print("skonfig", __version__)
         return True
+    from skonfig.configuration import bootstrap
```

Comment:
I'm not sure where we should call it. My initial thought was to do it in `cdist.configuration.get()`, but it doesn't get called until `skonfig <host>` is actually attempted. Second idea was to add explicit flag `--bootstrap`, which also will git clone sets to `~/.skonfig/set` (with y/N prompt), if there aren't any detected from `conf_dirs`. I don want to keep it simple, so maybe that's it?

Reply:
I'm not a big fan of options which change a program's function completely (like `-d` or `--bootstrap`).

If you feel like a completely separate utility (which could possibly also manage the installed sets) would be too unobvious for new users I think this is a good middle ground.